cmd/minishift/cmd/start.go

```diff
@@ -231,6 +232,35 @@ func updateNoProxyForDocker() string {
 	return "localhost,127.0.0.1,172.30.1.1"
 }
 
+//Update RegistrationProxyParameters with proxy information
+func SetRegistrationProxyParameters() {
+	var proxyUri string
+
+	if viper.IsSet("https-proxy") {
+		proxyUri = viper.GetString(httpsProxy)
+	} else if viper.IsSet("http-proxy") {
+		proxyUri = viper.GetString(httpProxy)
+	}
+
+	if proxyUri != "" {
+		proxyUriSlice, err := minishiftUtil.ParseProxyUri(proxyUri)
```

Comment:
proxyUriSlice could have been a structure. If you also think that I would return a structure , I can do that.